activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -171,6 +173,10 @@ def supports_explain?
         true
       end
 
+      def supports_foreign_keys?
+        true
+      end
+
```

Comment:
Okay. Actually, better yet, I'll have it ask SQLite with a PRAGMA whether or not foreign key support is working; it is possible for even recent versions of sqlite to be compiled with foreign key support deliberately disabled, though I don't know why anyone would do that.